build/docker/nginx/default.conf

```diff
@@ -9,6 +10,7 @@ server {
   location ~ /api/v[0-9]+/(users/)?websocket$ {
     proxy_set_header Upgrade $http_upgrade;
     proxy_set_header Connection "upgrade";
+	proxy_http_version 1.1;
```

Comment:
is there an extra space here?

Reply:
Tab vs space.  Not sure why the file is not tabbed.  Fixed.